workspace/build_target_test.go

```diff
@@ -0,0 +1,114 @@
+package workspace
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/yourbase/yb/runtime"
+)
+
+func TestBuildTarget_EnvironmentVariables(t *testing.T) {
+	type fields struct {
+		Name        string
+		Tools       []string
+		Commands    []string
+		Environment []string
+	}
+	type args struct {
+		data runtime.RuntimeEnvironmentData
+	}
+	// Sets up dotenv
+	cwd, err := filepath.Abs(".")
```

Comment:
nit: this can made more direct with `os.Getwd`:

```suggestion
	cwd, err := os.Getwd()
```